CoffeeEngine/src/CoffeeEngine/ParticleManager/ParticleManager.cpp

```diff
@@ -185,28 +235,32 @@ namespace Coffee
             glm::vec3 newSize;
             if (sizeOverLifeTimeSeparateAxes)
             {
-                newSize.x = CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeX);
-                newSize.y = CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeY);
-                newSize.z = CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeZ);
+
+                newSize.x = CurveEditor::ScaleCurveValue(CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeX), -1, 1);
+
+                newSize.y = CurveEditor::ScaleCurveValue(CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeY), -1, 1);
+
+                newSize.y = CurveEditor::ScaleCurveValue(CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeZ), -1, 1);
```

Comment:
Same here, i presume that you want to modify the Z axis size, not the Y axis size once again.

```suggestion
                newSize.z = CurveEditor::ScaleCurveValue(CurveEditor::GetCurveValue(normalizedLife, sizeOverLifetimeZ), -1, 1);
```

Reply:
The same as the other review.